projects/librdkafka.cmake

```diff
@@ -12,14 +12,15 @@ set(PRO_LIBRDKAFKA
   DESC "librdkafka is a C library implementation of the Apache Kafka protocol, containing both Producer and Consumer support -- [windows-only patch](../patches/librdkafka-windows.patch)"
   REPO "repo" ${REPO} "distributePro fork of librdkafka repo on github"
   VER ${VER}
-  GIT_ORIGIN git://github.com/distributepro/librdkafka.git
+  GIT_ORIGIN git://github.com/edenhill/librdkafka.git
   GIT_UPSTREAM git://github.com/edenhill/librdkafka.git
-  GIT_TAG xp-${VER} # what to 'git checkout'
+  GIT_TAG ${VER} # what to 'git checkout'
   GIT_REF ${VER} # create path from this tag to 'git checkout'
   DLURL ${REPO_UPSTREAM}/archive/${VER}.tar.gz
-  DLMD5 8e5685baa01554108ae8c8e9c97dc495
+  https://github.com/edenhill/librdkafka/archive/v1.5.3.tar.gz
+  DLMD5 17017d9bdaf1398087d1f0dcad2e5cc7
   DLNAME librdkafka-${VER}.tar.gz
-  PATCH ${PATCH_DIR}/librdkafka.patch
+  #PATCH ${PATCH_DIR}/librdkafka.patch
```

Comment:
I want to leave the patch until more thorough testing with builds that utilize ovsrpro

Reply:
We might need to create a new patch. The current patch

- Sets the debug prefix so we can install release and debug .a files.
- Sets where to install the targets so externpro can create the package.
- Adds rt to the link dependencies; I don't know if we still need this.

I say "create a new patch" because src-cpp/CMakeLists.txt has probably changed, so the current patch probably can't be applied.